skins/oasis/js/privacyUpdateNotification.js

```diff
@@ -5,7 +5,7 @@ $(function() {
         var notification = new BannerNotification(message).show();
 
         notification.onClose(function() {
-            window.Wikia.Cookies.set('dismissed-privacy-notification', true);
+            window.Wikia.Cookies.set('dismissed-privacy-notification', true, {domain: 'fandom.com'});
```

Comment:
I think this should be `{domain: '.fandom.com', path: '/'}` instead.

Reply:
I agree with V but double check the GDPR lib and use that logic

Reply:
Added path, but don't need the leading dot.

> Contrary to earlier specifications, leading dots in domain names (.example.com) are ignored.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie